swap/src/main.rs

```diff
@@ -209,6 +210,71 @@ async fn main() -> Result<()> {
             tokio::spawn(async move { event_loop.run().await });
             bob::run(swap).await?;
         }
+        Command::Cancel(Cancel::BuyXmr {
+            swap_id,
+            alice_peer_id,
+            alice_addr,
+            config,
+            force,
+        }) => {
+            // TODO: Optimization: Only init the Bitcoin wallet, Monero wallet unnecessary
+            let (bitcoin_wallet, monero_wallet) =
+                init_wallets(config.path, bitcoin_network, monero_network).await?;
+
+            let bob_factory = Builder::new(
+                seed,
+                db_path,
+                swap_id,
+                Arc::new(bitcoin_wallet),
+                Arc::new(monero_wallet),
+                alice_addr,
+                alice_peer_id,
+                execution_params,
+            );
+            let (swap, event_loop) = bob_factory.build().await?;
+
+            tokio::spawn(async move { event_loop.run().await });
+
+            match bob::cancel(swap.swap_id, swap.state, swap.bitcoin_wallet, swap.db, force).await? {
+                Ok((txid, _)) => { info!("Cancel transaction successfully published with id {}", txid)},
+                Err(CancelError::CancelTimelockNotExpiredYet) => {info!("The Cancel Transaction cannot be published yet, because the timelock has not expired. Please try again later.")},
```

Comment:
```suggestion
                Err(CancelError::CancelTimelockNotExpiredYet) => {error!("The Cancel Transaction cannot be published yet, because the timelock has not expired. Please try again later.")},
```
Did you run the formatter?

Reply:
Yes, I did run the formatter (otherwise CI `static_analysis` check would fail). - it did not pick the long lines up. Only after adding a linebreak manually in one of the statements and running the formatter again it picked it up and formatted the whole block. Strange :]